src/styles/index.js

```diff
@@ -109,3 +109,27 @@ export const Dot = styled.span`
 export const FallbackSpan = styled.span`
   opacity: 0.5;
 `;
+
+export const UpgradeNoticeStyles = ({
+  textDecoration = 'underline',
+} = {}) => props => css`
+  color: ${props.theme.colors.gold};
+  display: inline-block;
+  span {
+    white-space: normal !important;
+    display: unset;
+    font-size: 16px;
+    font-weight: 300;
+    &:hover {
+      text-decoration: ${textDecoration};
+    }
+  }
+  &:before {
+    display: inline-block;
+    content: '\f019';
```

Comment:
Is this the warning icon?

This is kinda cool, but at the same time also more cryptic to know what it is.

The span with `fa fa-warning` was pretty clear what it does... maybe we should extract that into a 'warning icon' component, or maybe add some comments here about what `\f019` means.

Reply:
I believe that's the unicode character code of the icon in the font-awesome font. As you'd have `\u26be` for a ⚾️emoji, for example.

There's a cheatsheet here
https://fontawesome.com/cheatsheet